localstack/testing/pytest/validation_tracking.py

```diff
@@ -42,7 +42,7 @@ def find_validation_data_for_item(item: pytest.Item) -> Optional[dict]:
 
 def record_passed_validation(item: pytest.Item, timestamp: Optional[datetime.datetime] = None):
     base_path = os.path.join(item.fspath.dirname, item.fspath.purebasename)
-    with open(f"{base_path}.validation.json", "w+") as fd:
+    with open(f"{base_path}.validation.json", "r+") as fd:
```

Comment:
oof, good catch... I only tried it with one test and totally missed that one. Sorry 😞